jcache/embedded/src/test/java/org/infinispan/jcache/JCacheLoaderAdapterTest.java

```diff
@@ -0,0 +1,81 @@
+package org.infinispan.jcache;
+
+import org.infinispan.jcache.embedded.JCacheLoaderAdapter;
+import org.infinispan.jcache.util.InMemoryJCacheLoader;
+import org.infinispan.marshall.TestObjectStreamMarshaller;
+import org.infinispan.marshall.core.MarshalledEntry;
+import org.infinispan.marshall.core.MarshalledEntryFactory;
+import org.infinispan.marshall.core.MarshalledEntryFactoryImpl;
+import org.infinispan.persistence.DummyInitializationContext;
+import org.infinispan.persistence.spi.InitializationContext;
+import org.infinispan.test.AbstractInfinispanTest;
+import org.infinispan.util.DefaultTimeService;
+import org.infinispan.util.TimeService;
+import org.testng.annotations.*;
+
+import static org.testng.AssertJUnit.*;
```

Comment:
Can you replace the `*` imports please. At the moment these imports cause checkstyle to fail